data/issues.py

```diff
@@ -0,0 +1,89 @@
+import logging
+
+import requests
+
+from data.models import (
+    Issue,
+    Label,
+    )
+from data.newcomers import get_newcomers
+from data.models import Contributor
+from data.web import web_url
+from community.git import get_org_name
+
+
+def fetch_issues(hoster):
+    """
+    Get issues opened by newcomers
+
+    :param hoster: a string representing hoster, e.g. 'github'
+    :return: a json of issues data
+    """
+    logger = logging.getLogger(__name__)
+    if hoster == 'github':
+        IMPORT_URL = web_url + 'issues/github/all'
+    elif hoster == 'gitlab':
+        IMPORT_URL = web_url + 'issues/github/all'
+    else:
+        IMPORT_URL = 'https://pastebin.com/raw/X9a6D5ym'
```

Comment:
what is this dataset?  'test'?  if so, the `else:` should be `elif hoster == 'test':`

and add a comment about this